### DISCUSSION TITLE
Tracking actors and directors upcoming projects

### DISCUSSION THREAD

**Discussion by IO-B** [IgnisDa/ryot#707] [Ideas]
I am not sure if this is already a feature and i just cant find it.
I do use the "monitor a person" feature. But it would be great to have an option to also see upcoming projects. Maybe even show those in the calendar. 
Also i really like the way "letterboxd" visualizes projects a director or actor worked on. 
A nice little feature is their percentage/number of watched projects per actor/director.

![grafik](https://github.com/IgnisDa/ryot/assets/83821430/e033bcd4-f74a-4600-876f-a5ce32db6cf3)

**Comment by IgnisDa** [comment]
Hmmm that's interesting. Please convert this discussion into an issue and I'll look into implementing it later.
